types/setting.go

```diff
@@ -491,6 +501,9 @@ var (
 		Required:    true,
 		ReadOnly:    false,
 		Default:     "100",
+		ValueIntRange: map[string]int{
+			ValueIntRangeMinimumKey: 0,
```

Comment:
Can the Default be 0? If so, what will happen when Default is 0?

Reply:
Yes it can be "0"
I define the value range by referring our original code
```
	case SettingNameStorageOverProvisioningPercentage:
		if _, err := strconv.Atoi(value); err != nil {
			return errors.Wrapf(err, "value %v is not a number", value)
		}
		// additional check whether over provisioning percentage is positive
		value, err := util.ConvertSize(value)
		if err != nil {
			return errors.Wrapf(err, "failed to parse %v as size", value)
		}
		if value < 0 {
			return fmt.Errorf("value %v should be positive", value)
		}
```

For this case, we only invalid the setting when it is < 0 , thats why I define the minimum as "0"

Other case would be the same.
But yes, it is a chance for us to review if the value range is correct.